tests/test_views.py

```diff
@@ -2160,8 +2162,8 @@ def test_claim_user_already_registered_redirects_to_claim_user_registered(self,
     @mock.patch('website.project.views.contributor.send_claim_email')
     def test_claim_user_already_registered_secondary_email_redirects_to_claim_user_registered(self, claim_email):
         name = fake.name()
-        email = fake.email()
-        secondary_email = fake.email()
+        email = fake_email()
+        secondary_email = fake_email()
```

Comment:
It would be worth checking here that the two emails are definitely different and, if not, delaying briefly and trying again. And/or failing with an assert that says that the processor speed is too fast for the clock resolution and that there may be similar problems elsewhere, possibly with a link to this ticket/pr.